scripts/data_overlap/compute_data_overlap_metrics.py

```diff
@@ -106,6 +107,20 @@ def create_ngram_index(
                         ngram_index[n][reference_ngram].add(
                             EntryDataOverlapKey(stats_key=stats_key, instance_id=id, part=PART_REF)
                         )
+
+                # compute intersection ngrams [defined as n-grams that occur between instance and reference]
```

Comment:
Not clear what intersection means exactly or "between instance and reference"...can this be defined a bit more precisely in natural language?

Reply:
Yep the example I give in the PR description is:
input = ["is 2+2 4 true or false"] reference = ["true"], the intersection is the 5-gram ["4 true or false true"] (which is formed from the input 4-gram [4 true or false] and the reference 1-gram [true]) I'll reword the comment